docs/source/features/search/index.rst

```diff
@@ -0,0 +1,15 @@
+..
+ Copyright 2021 Graviti. Licensed under MIT License.
+
+########
+ Search
+########
+
+TensorBay supports dataset search.
```

Comment:
add more details. like
TensorBay support building filters to search for data at multiple levels, such as segment, data size ....